src/main/docker/Dockerfile

```diff
@@ -12,22 +12,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM java:8u92-jre-alpine
+FROM java:8u111-jre-alpine
```

Comment:
The "java" base image is deprecated.  Please use the opkenjdk base image instead.